test/src/sdk-harness/test_projects/call_frames/mod.rs

```diff
@@ -41,14 +41,30 @@ async fn can_get_code_size() {
 async fn can_get_first_param() {
     let (instance, _id) = get_call_frames_instance().await;
     let result = instance.get_first_param().call().await.unwrap();
-    assert!(is_within_range(result.value));
+    // The hash of "get_first_param" is 0x35fd8826
```

Comment:
Can this be fetched and then converted to a value automatically instead of hardcoding a magic number with a comment?